definitions/json/CaseEvent/CaseEvent.json

```
@@ -1016,5 +1016,48 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "ShowSummary": "N"
+  },
+  {
+    "CaseTypeID": "ET_Scotland",
+    "ID": "claimantTSE",
+    "Name": "Make an Application",
+    "Description": "Claimant Tell Something Else",
+    "PreConditionState(s)": "Submitted;Vetted;Rejected;Accepted",
```

They should also be able to make applications when it's closed
```suggestion
    "PreConditionState(s)": "Submitted;Vetted;Rejected;Accepted;Closed",
```